create-kafka-queues/src/main/resources/values.yaml

```diff
@@ -17,6 +17,26 @@ replicaCount: 1
 service:
   port: 80
 
+kafkaCreateTopics:
+  - name: palisade
+    replicationFactor: 1
+    partitions: 1
+  - name: user
+    replicationFactor: 1
+    partitions: 1
+  - name: resource
+    replicationFactor: 1
+    partitions: 1
+  - name: policy
+    replicationFactor: 1
+    partitions: 1
+  - name: query-scope
+    replicationFactor: 1
+    partitions: 1
+  - name: results-controller
+    replicationFactor: 1
+    partitions: 1
+
 image:
```

Comment:
missing codeRelease

Reply:
it's not a service artifact - doesn't require it